appengine/flexible/scipy/requirements.txt

```
@@ -2,9 +2,9 @@ Flask==3.0.3
 gunicorn==23.0.0
 imageio==2.35.1; python_version == '3.8'
 imageio==2.36.1; python_version >= '3.9'
-numpy==2.0.0; python_version > '3.9'
-numpy==1.26.4; python_version == '3.9'
-numpy==1.24.4; python_version == '3.8'
+numpy==2.2.4; python_version > '3.9'
+numpy==2.2.4; python_version == '3.9'
+numpy==2.2.4; python_version == '3.8'
```

```suggestion
```suggestion
numpy==2.2.4; python_version > '3.9'
numpy==1.26.4; python_version == '3.9'
numpy==1.24.4; python_version == '3.8'
```
```